pkgs/servers/geospatial/pg_tileserv/fix-go-version-error.patch

```diff
@@ -0,0 +1,61 @@
+diff --git i/go.mod w/go.mod
+index 8ee3eec..dadae0d 100644
+--- i/go.mod
++++ w/go.mod
+@@ -1,6 +1,6 @@
+ module github.com/CrunchyData/pg_tileserv
+ 
+-go 1.15
++go 1.17
```

Comment:
Would you mind trying to upstream this?

Reply:
https://github.com/CrunchyData/pg_tileserv/pull/182

Reply:
Thank you! 
One last thing: can you use fetchpatch with https://github.com/CrunchyData/pg_tileserv/commit/d8e01469344895267ead9fa35ee7bdb8f966a710 with a comment to that PR. fetchpatch is preffered in nixpkgs to avoid increasing the size of the repository.